src/autosuspend/checks/activity.py

```diff
@@ -484,6 +485,16 @@ def check(self) -> Optional[str]:
         return None
 
 
+def _list_logind_sessions(logger: logging.Logger) -> Iterable[Tuple[str, dict]]:
+    try:
+        return list_logind_sessions()
+    except LogindDBusException as error:
+        logger.warning(
```

Comment:
I think it would be ok to remove the logging here. TemporaryCheckErrors are logged by the surrounding infrastructure in any case.

Reply:
Removed logging.